website/cue/reference/components/sources/opentelemetry.cue

```diff
@@ -0,0 +1,209 @@
+package metadata
+
+components: sources: opentelemetry: {
+	_port: 6788
+
+	title: "Opentelemetry"
+
+	description: """
+		Collect opentelemetry data over grpc (currently, only log is supported).
+		"""
+
+	classes: {
+		commonly_used: false
+		delivery:      "at_least_once"
+		deployment_roles: ["aggregator"]
+		development:   "beta"
+		egress_method: "stream"
+		stateful:      false
+	}
+
+	features: {
+		acknowledgements: true
+		multiline: enabled: false
+		receive: {
+			from: {
+				service: services.opentelemetry
+
+				interface: socket: {
+					direction: "incoming"
+					port:      _port
+					protocols: ["tcp"]
+					ssl: "optional"
+				}
+			}
+			receive_buffer_bytes: enabled: false
+			keepalive: enabled:            true
+			tls: {
+				enabled:                true
+				can_verify_certificate: true
+				enabled_default:        false
+			}
+		}
+	}
+
+	support: {
+		requirements: []
+		warnings: []
+		notices: []
+	}
+
+	installation: {
+		platform_name: null
+	}
+
+	configuration: {
+		acknowledgements: configuration._source_acknowledgements
+		address: {
+			description: """
+				The grpc address to listen for connections on. It _must_ include a port.
+				"""
+			required: true
+			type: string: {
+				examples: ["0.0.0.0:\(_port)"]
+			}
+		}
+		multiple_outputs: {
+			common: false
+			description: """
+				If this setting is set to `true` logs, metrics and traces will be sent to different ouputs. For a source
+				component named `agent` the received logs, metrics, and traces can then be accessed by specifying
+				`agent.logs`, `agent.metrics`, and `agent.traces`, respectively, as the input to another component.
+				"""
+			required: false
+			type: bool: default: false
+		}
+	}
+
+	outputs: [
+		{
+			name: components._default_output.name
+			description: """
+				Default output stream of the component. Use this component's ID as an input to downstream transforms and sinks. Only active if [multiple_outputs](#multiple_outputs) is disabled.
+				"""
+		},
+		{
+			name: "logs"
+			description: """
+				If [multiple_outputs](#multiple_outputs) is enabled, received log events will go to this output stream. Use `<component_id>.logs` as an input to downstream transforms and sinks.
+				"""
+		},
+	]
+
+	output: {
+		logs: event: {
+			description: "An individual event from a batch of events received through a grpc request sent by opentemetry sdk"
```

Comment:
I'm going to take another pass through the docs and update all the descriptions of the event fields to be complete sentences - just wanted to drop the current feedback I had.